quoter.lua

```diff
@@ -18,6 +17,10 @@ function preRune(bp, r)
 	if bp.Cursor:HasSelection() == false then
 		return true
 	end
+	
+	-- try using user settings, or use "default"
+	local quotePairs = modes[config.GetGlobalOption("quoter.mode")]
```

Comment:
```suggestion
	local quotePairs = modes[bp.Buf.Settings["quoter.mode"]]
```

I think it would also be better to display an error in the infobar if the set mode is not in the `modes` table.

Reply:
Is there a callback for when a set command is used? The best way I found to achieve this is by checking if the mode is valid when applying quotes.

Reply:
There is none. `onBufferOptionChanged` was added after micro 2.0.14 is released, but I don't think there will be an update in several months.